### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by khalibartan**
Ping @ankurankan

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,22 +5,27 @@ pgmpy
...
-- Numpy 1.9.2
-- Cython 0.21
-- Pandas 0.15.1
+Pgmpy has following non optional dependencies
+- Python 2.7 or higher versions, Python 3 or higher versions
+- NetworkX 1.9.1 or higher
+- Scipy 0.12.1 or higher
+- Numpy 1.9.2 or higher
+- Cython 0.21 or higher
+- Pandas 0.15.1 or higher ( except 0.17.0 )
```
Comment (ankurankan) [inline README.md]: @khalibartan Don't say higher, just the exact versions that we have. Might not work with all the versions.

File: README.md [hunk 2]
```diff
@@ -30,8 +35,32 @@ pip install -r requirements.txt
...
+To install pgmpy first you have to install the dependencies.
+To install the dependencies first go to the pgmpy directory and then run the following commands
+```
+$ cd /path/to/pgmpy
+```
+In the directory run either of the following
+
+Using pip
+```
+$ pip install -r requirements.txt
```
Comment (ankurankan) [inline README.md:46]: @khalibartan Repeating how to install requirements. This is already mentioned above.

File: README.md [hunk 3]
```diff
@@ -39,104 +68,135 @@ to build a python extension module:
 sudo python3 setup.py install
 ```
 
-Development:
+If you face any problems don't open an issue, reach us at our mailing list or gitter channel (mentioned above)
```
Comment (ankurankan) [inline README.md]: @khalibartan No problem in opening issues.

File: README.md [hunk 4]
```diff
@@ -39,104 +68,135 @@ to build a python extension module:
...
+You can check the latest sources from our github repository 
+use the command:
 
     git clone https://github.com/pgmpy/pgmpy
 
 Contributing
 ------------
+We accept contribution in terms of issues and pull requests.
+
+Before opening a issue, please have a look that if we already have that issue opened. If not than open a new issue.
```
Comment (ankurankan) [inline README.md]: @khalibartan Doesn't sound very welcoming.

File: README.md [hunk 5]
```diff
@@ -39,104 +68,135 @@ to build a python extension module:
...
 
-Before opening a Pull Request, have a look at the
-full Contributing page to make sure your code complies
-with our guidelines - 
-https://github.com/pgmpy/pgmpy/blob/dev/Contributing.md
+Before opening a pull request , have look at our [contributing guide](
+https://github.com/pgmpy/pgmpy/blob/dev/Contributing.md)
+
+please adhere to the points mentioned here, it makes review process difficult if you don't follow them.
```
Comment (ankurankan) [inline README.md]: @khalibartan Same here. You need to use a much softer tone.

**Comment by khalibartan** [comment]
@ankurankan Sorry, for that rude sections. Here the new commit I think is more friendly  :)

**Pull request merged**